core/backend/src/IModelDb.ts

```diff
@@ -1338,6 +1338,21 @@ export abstract class IModelDb extends IModel {
       }
     });
   }
+
+  /**
+   * For cases where you need to preserve exact [[Code]]s between databases,
+   * you can set the codeValueBehavior to "exact". The default is "trim-unicode-whitespace".
+   * Code Values should be trimmed, this is primiarly for compatibility with iModels that
+   * failed to trim their codes.
+   * @beta
```

Comment:
I'm concerned that this is `beta` and doesn't make clear that it only affects code values created in native code. Almost nobody using our TS APIs is going to understand the ramifications of that.

Reply:
Would you prefer I make it internal but note that the transformer packages use it?

Reply:
Do you expect users to know - or realize they should know - that an iModel they're running the transformer on may suffer this bug and need to set this property?
Or will you just make the transformer always set this to "exact"?

Reply:
transformer always sets this to exact

Reply:
If anyone takes a code value read out of the native db and passes it to the TS `Code` constructor, new fun bugs will ensue. See suggested change below.